### PR TITLE
Belt harness

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds a belt harness that holds special pouches that you can't get because I have no clue how to add it to the game
the sprites are placeholders from mio/escalation repo

## Why It's Good For The Game
i have no clue I just felt like making this

## Changelog
:cl:
add: Added new type of harness pouch and harness for it to go on. Adds a harness admin vendorblender.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -37. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Hughgent** [comment]
Hey, you probably want this proc `remove_from_storage` to open the inventory of your storage pouches.

**Comment by jroinc** [comment]
This is an abomination against god and man.

It'll fit right in.

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -637,3 +637,84 @@
...
+	storage_slots = 1
+	draw_mode = 0
+	allow_drawing_method = TRUE
+
+/obj/item/storage/pouch/harness/utility
+	name = "general harness pouch"
+	desc = "A general purpose pouch used to carry two small items. Designed to be clipped to the M-210 harness. Too big to fit in a pocket."
+	storage_slots = 2
+	icon_state = "belt_utility"
+	draw_mode = 0
```
Comment (psykzz) [inline code/game/objects/items/storage/pouch.dm:659]: No need to duplicate this everywhere

File: code/game/objects/items/storage/pouch.dm [hunk 2]
```diff
@@ -637,3 +637,84 @@
...
+
+/obj/item/storage/pouch/harness/utility/big
+	name = "general harness pouch"
+	desc = "A general purpose pouch used to carry a single large item. Designed to be clipped to the M-210 harness. Too big to fit in a pocket."
+	storage_slots = 1
+	max_w_class = 3
```
Comment (psykzz) [inline code/game/objects/items/storage/pouch.dm:665]: make sure to use the weight class defines

**Comment by Pariah919** [comment]
random ideas electric boogalo

**Pull request closed without merging**